plugins/editors/tinymce/src/PluginTraits/DisplayTrait.php

```diff
@@ -459,6 +459,10 @@ public function display(string $name, string $content = '', array $attributes =
                 'branding'  => false,
                 'promotion' => false,
 
+                // Hardened security
+                'sandbox_iframes' => true,
+                'convert_unsafe_embeds' => true,
```

Comment:
```suggestion
                'sandbox_iframes'       => true,
                'convert_unsafe_embeds' => true,
```
Fix PHPCS failing in Drone.